hack/dev-mounter/main.go

```diff
@@ -61,10 +62,33 @@ func newCommand() *cobra.Command {
 					log.Warnf("Failed to create directory: %v", err)
 					return
 				}
+				// Remove files that do not exist in ConfigMap anymore
+				err = filepath.Walk(destPath, func(path string, info os.FileInfo, err error) error {
+					if info.IsDir() {
+						return nil
+					}
+					if err != nil {
+						log.Warnf("Error walking path %s: %v", path, err)
+					}
+					p := filepath.Base(path)
+					if _, ok := cm.Data[p]; !ok {
+						log.Infof("Removing file '%s'", path)
+						err := os.Remove(path)
+						if err != nil {
+							log.Warnf("Failed to remove file %s: %v", path, err)
+						}
+					}
+					return nil
+				})
+				if err != nil {
+					log.Fatalf("Error: %v", err)
+				}
+				// Create or update files that are specified in ConfigMap
```

Comment:
@alexmt FYI, I've also added functionality to delete files from the file system which do not exist in the config map anymore. This makes local/manual testing possible.